tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -3458,6 +3458,64 @@ def test_filtering_by_org_id_is_not_compatible(self):
         )
         assert response.status_code == 400, response.content
 
+    @mock.patch("sentry.snuba.transactions.query")
+    def test_mep_uses_transactions_dataset_when_fallback_is_specified(
```

Comment:
I propose we add extensive test cases now - ie, cover all the test cases that the `discover` arm catches. because once that's removed we might have holes in the tests.

Reply:
We should also include tests for `timeseries_query`